### DISCUSSION TITLE
So far I am loving this!

### DISCUSSION THREAD

**Discussion by danielbrunt57** [emporia-vue-local/esphome#45] [General]
This is far better than my CurrentCost Energy Monitor! I just bought the Vue2 with 8 clamps but have ordered another 8 clamps already. My panel has 24 circuits so 16 monitored is better than 8.  I tried all afternoon to get this project installed via ESPHome in HA on Odroid but finally conceded and installed ESPHome on Windows 11.  Original firmware saved and then reflashed with new yaml/project no problem.  I hope HA ESPHome on Odroid catches up to Windows, soon!

**Comment by clowrey** [comment]
It's all to do with the ARM toolchain not compatible yet.. can't use it on raspberry pi either yet. But it's being worked on will eventually.. I switched my whole HA install to x86 VM to get this working easily 😂 after wasting many hours trying to fix the arm part..

**Reply by danielbrunt57** [reply]
Yeah, I eventually realized I was banging my head against a brick wall 🧱🤕!  I just wish I hadn't wasted an entire afternoon trying to fix something beyond my expertise and had gone straight to plan 'B' 🤔!!  I'll use ESPHome on my PC to make any updates to it I might need until ARM catches up...
Depending on the complexity of your HA, the x86 VM is probably working better for you.  My HA started 18 mos ago on a very old Intel NUC but I splurged 9 mos ago for the HA "Blue" ODROID-N2+.
